src/main/java/seedu/address/model/owner/Address.java

```diff
@@ -9,13 +9,13 @@
  */
 public class Address {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Addresses must be of length 3";
```

Comment:
clearer to say "Address must be at least 3 characters long" something like that?